scenarios/ci/pitr.lua

```diff
@@ -0,0 +1,88 @@
+require("pg_simple")
+
+function main(argv)
+	if fs.is_directory("backups") then
+		warning("Deleting old stormweaver run backups")
+		fs.delete_directory("backups")
+	end
+
+	if fs.is_directory("archive") then
+		warning("Deleting old stormweaver run archive")
+		fs.delete_directory("archive")
+	end
+
+	os.execute("mkdir archive")
+
+	simple_pg_single(argv, function(args)
+		pgm:basebackup(1, "-D", "backups/backup_0", "-U", "stormweaver", "-c", "fast")
+
+		times = {}
+
+		for workloadIdx = 1, tonumber(args["repeat"]) + 1 do
+			t1:run()
+			t1:wait_completion()
+			pgm:get(1, 10)
+
+			sleep(3000)
+
+			t = os.time()
+
+			w = pgm.primaryNode:make_worker("verification")
+			w:calculate_database_checksums("backups/" .. t .. ".checksum")
+
+			table.insert(times, t)
+
+			sleep(3000)
+		end
+
+		fs.delete_directory("archive-copy")
+		os.execute("cp -r archive archive-copy")
```

Comment:
Why do we need to delete it here and into the loop?

Reply:
It's different: here we delete archive-copy, in case it is a leftover from a previous execution. I guess this could go to the top of the file too, where we delete leftover archive and backups too.

In the loop we delete the archive, which we copy fresly every time.